frontend/src/components/Filters/StatefulFilters.tsx

```diff
@@ -271,6 +271,7 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
           variant={SelectVariant.typeahead}
           isExpanded={this.state.isExpanded}
           aria-label="filter_select_value"
+          data-cy="filter-value"
```

Comment:
I think we are annotating here for the same use case implemented using the "aria-*" in another test:
![image](https://user-images.githubusercontent.com/1662329/164204758-288e2b2f-99bd-40fe-8103-037283d87010.png)

I don't really mind one way or another, but it shouldn't diverge too much IMO.

In this case, the "data-cy" looks clear

Reply:
Maybe I can add a PR with the changes in the existing tests and try to find a sync point between all folks.
I guess I've added more text in the comments rather that the change in the naming :).

Reply:
Sure I'm happy to change one way or the other. I don't have a preference.